src/Utils.php

```diff
@@ -358,14 +358,24 @@ public static function tryFopen($filename, $mode)
         $ex = null;
         set_error_handler(function () use ($filename, $mode, &$ex) {
             $ex = new \RuntimeException(sprintf(
-                'Unable to open %s using mode %s: %s',
+                'Unable to open "%s" using mode "%s": %s',
                 $filename,
                 $mode,
                 func_get_args()[1]
             ));
         });
 
-        $handle = fopen($filename, $mode);
+        try {
+            $handle = fopen($filename, $mode);
+        } catch (\Throwable $e) {
```

Comment:
This will not work on PHP 5.4. Right?

Reply:
PHP 5 only ever raises errors, and never "throws" anything. :)

Reply:
Yeah, but does `Throwable` exist in PHP 5.4?

Reply:
That's fine. Classes in a catch block don't trigger the autoloader.

Reply:
True, But we will never catch an exception in PHP 5.4. Right?

Reply:
Yes. That's no problem because:

> `fopen` on PHP 5 only ever raises errors, and never "throws" anything.

Reply:
Oh, sorry. I understand now. 

Thank you.